src/controller/java/src/chip/clusterinfo/ResponseValueInfo.java

```diff
@@ -0,0 +1,14 @@
+package chip.clusterinfo;
+
+/** CommandParameterInfo captures the name and type of a parameter */
```

Comment:
Update comment? Also, I think we could just name it `CommandResponseInfo`, `ValueInfo` seems redundant.